accepted/2025/prune-package-reference-rollout.md

```diff
@@ -0,0 +1,70 @@
+# PrunePackageReference in .NET 10
+
+- Nikolche Kolev <https://github.com/nkolev92>
+- [#NuGet/Client.Engineering#3110](https://github.com/NuGet/Client.Engineering/issues/3110)
+
+## Summary
+
+`NuGetAudit` scans the dependencies in your package graph for vulnerabilities, reporting vulnerabilities for every package that ended up in your resulting graph.
+
+[`PrunePackageReference`](https://github.com/NuGet/Home/blob/dev/accepted/2024/prune-package-reference.md) allows NuGet to skip platform packages that are already available within the framework that the project is building.
+For example, if your project, targetting .NET 9 has a transitive dependency to System.Text.Json 8.0.4, which is vulnerable, enabling pruning will allow NuGet to skip it. This package will not be downloaded and it will not appear in the dependencies.
```

Comment:
nit: The word "skip" seems too vague, I thought it would be better to really call out what the implementation would do.

```suggestion
For example, if your project, targetting .NET 9 has a transitive dependency to System.Text.Json 8.0.4, which is vulnerable, enabling pruning will allow NuGet to prune it from the defined list of dependencies during graph resolution. This package will not be downloaded and it will not appear in the resolved dependency graph.
```